internal/gcsx/bucket_manager.go

```diff
@@ -82,19 +83,31 @@ type bucketManager struct {
 	stopGarbageCollecting func()
 }
 
-func NewBucketManager(config BucketConfig, storageHandle storage.StorageHandle) BucketManager {
+func createStatCacheLru(statCacheCapacity int) *lru.Cache {
+	var statCacheSize uint64
+	if statCacheCapacity > 0 {
+		// This conversion from capacity to size will not be needed when
+		/// StatCacheCapacity is replaced with StatCacheSizeMB
+		statCacheSize = uint64(statCacheCapacity) * metadata.StatCacheEntrySize()
+	} else if statCacheCapacity == -1 {
```

Comment:
We can live without supporting -1 for stat-cache-capacity flag because we currently don't support -1 and we are anyways going to deprecate these flags in future.

Reply:
Removed.